phpunit.xml

```diff
@@ -10,21 +10,9 @@
 	failOnWarning="true"
 	verbose="true">
 	<php>
-		<const
-			name="OVERRIDE_MYSQL_CONFIG"
-			value="true" />
-		<const
-			name="OVERRIDE_MYSQL_HOST"
-			value="127.0.0.1" />
-		<const
-			name="OVERRIDE_MYSQL_USER"
-			value="smr" />
-		<const
-			name="OVERRIDE_MYSQL_PASSWORD"
-			value="smr" />
-		<const
-			name="OVERRIDE_MYSQL_PORT"
-			value="3307" />
+		<env
+			name="MYSQL_CONFIG_ENVIRONMENT"
```

Comment:
Is the `MYSQL_CONFIG_ENVIRONMENT` variable being used anymore? It seems like no, since we're mounting `test.env` in the container as `.env`?